config/sara/jurisdiction_messages.yml

```diff
@@ -0,0 +1,28 @@
+'USA':
+  children:
+    'State 1':
+      messages:
```

Comment:
is there a reason for putting these in a new file instead of just adding another field to `jurisdictions.yml`?

Reply:
I went back and forth, but my thinking was to avoid cluttering `jurisdictions.yml`, and I thought it might be a little cleaner to have separate files for the separate rake tasks. But I do also see the benefit of not having an additional file that is expected to have the same hierarchy. What's your opinion?

Reply:
that makes sense, I think I would personally put it in `jurisdictions.yml` because that could prevent any problems with these files being out of sync or forgetting to run both rake tasks or running them out of order/etc, but it's not a big deal

Reply:
Thought about it some more and I think I'm going to leave it. The jurisdiction import task performs updates on threshold condition hashes for every jurisdiction no matter what even if unchanged, and so I like being able to manage the custom messages independently of that task.